frontend/cypress/integration/editor/editor.spec.js

```diff
@@ -1,16 +1,44 @@
-describe('When visiting the application, as a user', () => {
+import { v4 as uuid } from 'uuid'
+
+describe('When visiting the edit view page, as a user', () => {
   beforeEach(() => {
-    cy.visit(Cypress.env('HOST'))
+    cy.visit(Cypress.env('HOST') + '/edit')
   })
 
-  it('I can move to edit view page', () => {
-    cy.contains('Edit view').click()
+  it('I will be asked to log in when I am not logged in', () => {
     cy.url().should('include', '/edit')
+    cy.contains('Please log in!')
   })
-})
 
-describe('When visiting the edit view page, as a user', () => {
-  beforeEach(() => {
+  it('I can open the editor when I am logged in', () => {
+    
+    cy.visit(Cypress.env('HOST') + '/register')
+
+    const randomStr = uuid()
+    const username = randomStr.substr(0, 4)
+    const email = `${username}@test.com`
+    const password = 'testUserPass!111'
+
+    cy.get('#username').type(username)  
+    cy.get('#email').type(email)
+    cy.get('#password').type(password)
+    cy.get('#confirmPassword').type(password)  
+    cy.get('form button').click()
+
+    cy.wait(1000)
+
+    cy.visit(Cypress.env('HOST') + '/login')
```

Comment:
should we use cy.get('#loginButton').click() instead of cy.visist()? It would be more realistic situation because users navigate in our app using buttons and not by typing urls

Reply:
It would be a good idea when we have some unique identifiers for menu items.